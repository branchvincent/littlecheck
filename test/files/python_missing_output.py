# RUN: /usr/bin/python3 %s

import sys

print("First")
# CHECK: First

# CHECK: NotPrinted

print("Second")
# CHECK: Second
print("Third")
# CHECK: Third

print("1")
# CHECK: {{\d}}
print("2")
# CHECK: {{\d}}
print("3")
# CHECK: {{\d}}
print("4")
# CHECK: {{\d}}
print("5")
# CHECK: {{\d}}
print("6")
# CHECK: {{\d}}
print("6")
# CHECK: {{\d}}
print("foobar")
print("barbar")
# CHECK: {{barbar}}
print("6")
# CHECK: {{\d}}
print("6")
# CHECK: {{\d}}
print("6")
# CHECK: {{\d}}
print("6")
# CHECK: {{\d}}
print("6")
# CHECK: {{\d}}
print("6")
# CHECK: {{\d}}

print("Fourth")
