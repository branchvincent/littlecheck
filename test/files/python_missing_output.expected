Failure in python_missing_output.py:

  The CHECK on line 8 wants:
    NotPrinted

  which failed to match line stdout:2:
    Second

  Context:
    First
     <= nothing to match CHECK on line 8: NotPrinted
    Second
    Third
    1
    [...] from line 24 (stdout:8):
    5
    6
    6
    foobar <= no check matches this, previous check on line 28
    barbar
    6
    6
    [...] from line 39 (stdout:16):
    6
    6
    6
    Fourth <= no more checks

  when running command:
    /usr/bin/python python_missing_output.py
